apps/desktop-wallet/src/modals/AddressSweepModal.tsx

```diff
@@ -79,13 +79,12 @@ const AddressSweepModal = ({ sweepAddress, onClose, onSuccessfulSweep }: Address
         setFee(fees)
       } catch (e) {
         dispatch(transactionBuildFailed(getHumanReadableError(e, t('Error while building transaction'))))
-        posthog.capture('Error', { message: 'Building transaction' })
```

Comment:
I'll bring this back now that we have throttling and no fear of getting spammed again